pkgs/development/python-modules/refery/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+
+, poetry-core
+, pyyaml
+, colorama
+, junit-xml
+}:
+
+buildPythonPackage rec {
+  pname = "refery";
+  version = "2.1.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "daafdc841e2293db12dcf11bcc38605fbc347a774ee3af1cb23e60cd3a75f6a2";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pyyaml
+    colorama
+    junit-xml
+  ];
+
+  pythonImportCheck = [
+    "refery"
+  ];
+
+  disabled = pythonOlder "3.10";
+
+  # No tests yet
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Functional testing tool";
+    homepage = "https://github.com/RostanTabet/refery";
+    maintainers = with maintainers; [ rostan-t ];
```

```suggestion
		mainProgram = "refery";
    maintainers = with maintainers; [ rostan-t ];
```
setting mainProgram to the binary name is nice for things like `lib.getExe` or `nix run` so they don't have to guess.

---

Thanks! This is resolved by d04bb2d74f720f1ba1d78850e0300327be05d49f.